app/controllers/project_instances_controller.rb

```diff
@@ -71,6 +71,7 @@ def new
 
   def create
     @project_instance = ProjectInstance.new(project_instance_params)
+    @project_instance.populate_name(project_instance_params[:project_id], project_instance_params[:stage])
```

Comment:
at this stage @project_instance has the project_id and stage attributes set, so there's not much need passing them around as params - I think there's room to simplify the populate_name method so that it doesn't take any parameters